themes/finna2/templates/layout/layout.phtml

```diff
@@ -478,6 +478,19 @@ JS;
     <?php if ($finnaSurvey): ?>
       <?=$this->finnaSurvey()->render()?>
     <?php endif; ?>
+    <?php
+      // Show postLoginRoute in lightbox
+      if ($this->layout()->getTemplate() !== 'layout/lightbox') {
+        $session = new \Laminas\Session\Container('Followup');
```

Comment:
Ei voi tehdä näin, tästä puuttuu session container (vrt. FollowupFactory.php). Tämä kaikki pitäisi hakea controller-tasolla tai ainakin jossain view helperissä, joka hakee tiedon Followup-helperistä.

Reply:
Lisäsin view helpperin joka ohjaa retrieve-kutsut controller pluginille.